app/views/api_doc_view.php

```diff
@@ -67,7 +67,8 @@
 <pre><code class="json">{
     "title": "Blabla",
     "short_desc": "Blabla",
-    "long_desc": "Blabla"
+    "long_desc": "Blabla",
+    "whatsnew": "Blabla"
 }</code></pre>
 </p>
 
```

Comment:
Update the API doc view to remove all references to the google 'next' channel please.